db/wpdb.ts

```diff
@@ -110,7 +110,12 @@ class WPDB {
     }
 }
 
-export const singleton = new WPDB()
+let singleton: WPDB | undefined = undefined
+
+export function getSingleton(): WPDB {
+    if (!singleton) singleton = new WPDB()
```

Comment:
What does this change do?

Reply:
I'll take this out again - I thought the old code initialized a db connection as a side effect of being imported but [Matthieu made me realize that this is not the case](https://github.com/owid/owid-grapher/pull/1909).